src/tiffcomposite_int.hpp

```diff
@@ -397,7 +397,9 @@ namespace Exiv2 {
     //! Search key for TIFF mapping structures.
     struct TiffMappingInfo::Key {
         //! Constructor
-        Key(const std::string& m, uint32_t e, IfdId g) : m_(m), e_(e), g_(g) {}
+        Key(std::string m, uint32_t e, IfdId g) : m_(std::move(m)), e_(e), g_(g)
+        {
+        }
```

Comment:
```suggestion
        Key(std::string m, uint32_t e, IfdId g) : m_(std::move(m)), e_(e), g_(g){}
```

Reply:
git clang-format is what sets it to this.